requirements/edx/github.txt

```diff
@@ -55,6 +55,7 @@ git+https://github.com/edx/edx-lint.git@ed8c8d2a0267d4d42f43642d193e25f8bd575d9b
 -e git+https://github.com/edx/edx-reverification-block.git@a286e89c73e1b788e35ac5b08a54b71a9fa63cfd#egg=edx-reverification-block
 git+https://github.com/edx/ecommerce-api-client.git@1.1.0#egg=ecommerce-api-client==1.1.0
 -e git+https://github.com/edx/edx-user-state-client.git@64a8b603f42669bb7fdca03d364d4e8d3d6ad67d#egg=edx-user-state-client
+-e git+https://github.com/edx/edx-organizations.git@266ed317d4b4c860e6de013af2b323dbff019379#egg=edx-organizations
 
```

Comment:
Okay -- I will tag the latest edx-organizations changeset as release-2015-08-03 and update the branch